### ISSUE TITLE
Keyboard focus: dialogue box HP Statistics: tabbing in background

### ISSUE THREAD

**Issue by lizscuffell** [epimorphics/ukhpi#250]
This issue is related to #249 

Once the dialogue box has been opened for the location picker, the keyboard focus cycles through each option (apart from those in issue #249 ) and then proceeds to tab through the options in the background leaving the user unable to dismiss the modal. The keyboard focus needs to remain cycling through the options within the dialogue box until the dialogue box is dismissed.

**Comment by lizscuffell** [comment]
Complete
